modules/flatcar-linux/data/version.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+
+if [ ! "$(command -v jq)" ]; then
+  echo "jq is not installed."
+  exit 1
+fi
+
+eval "$(jq -r '@sh "FLATCAR_CHANNEL=\(.flatcar_channel) FLATCAR_VERSION=\(.flatcar_version) AWS_REGION=\(.aws_region)"')"
+
+if [ "$FLATCAR_VERSION" = "latest" ]; then
+  if [ "$AWS_REGION" != "cn-north-1" ] && [ "$AWS_REGION" != "cn-northwest-1" ]; then
```

Comment:
```suggestion
  if [ "$AWS_REGION" != cn-* ]; then
```

Reply:
Unfortunately regex in the shell does not work in that way but thank you anyway for the suggestion.

Reply:
fyi https://www.networkworld.com/article/2693361/unix-tip-using-bash-s-regular-expressions.html

Reply:
I didn't realise you're using sh here.